src/App.js

```diff
@@ -1,19 +1,16 @@
 import './App.scss';
 import Account from './components/account/account';
 import Grid from './components/common/grid'
-import Header from './components/header/header';
 import { Routes, Route } from "react-router-dom";
 import ScreenContainer from './components/common/screen-container';
 import Login from './components/login/login';
 import SignUp from './components/signup/sign-up';
 import routes from './constants/routes';
 
-//import SecondaryScreenContainer from './components/common/secondary-screen-container';
-
 function App() {
     return (
         <Grid>
-            <Header />
+            {/* <Header /> */}
```

Comment:
Better remove unused components. We always can re-create it later

Reply:
got it